### PR TITLE
raidboss: add alerts for Chimera in Cutter's Cry

### PR DESCRIPTION
**Pull request by jacob-keller**
Add basic alerts for the "Dragon's Voice" and "Ram's Voice" attacks of
the Chimera in Cutter's Cry.

These are straightforward alerts to warn the player to get in or get out
depending on if it is Dragon's Voice (A lightning donut AOE) or Ram's
Voice (An ice explosion in melee area).

Signed-off-by: Jacob Keller <jacob.keller@gmail.com>

### CODE REVIEW

File: ui/raidboss/data/02-arr/dungeon/cutters_cry.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+export type Data = RaidbossData;
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.CuttersCry,
+  triggers: [
+    {
+      id: 'Chimera Ram Voice',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '450', source: 'Chimera', capture: false }),
+      suppressSeconds: 0.1,
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/dungeon/cutters_cry.ts:16]: Are there really multiple starts using lines here that need to be suppressed?

Comment (jacob-keller) [reply ui/raidboss/data/02-arr/dungeon/cutters_cry.ts:16]: Oops, nope.

**Comment by jacob-keller** [comment]
Hmm. I just double checked things in the emulator again and it doesn't seem like it is working properly. Any ideas? It seems like a straight forward trigger...

**Comment by jacob-keller** [comment]
I've also checked a bunch of log files I have and don't see any other IDs for either Ram's Voice or Dragon's Voice when used by Chimera.

**Pull request merged**